### PR TITLE
Feature new component docs

### PR DESCRIPTION
**Pull request by blackbaud-brandonhare**

### CODE REVIEW

File: src/app/components/image/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <stache-code>imageSource</stache-code> — The path to the image to render.
+      </li>
+    </ul>
+  </stache-page-summary>
+
+  <stache-page-anchor>
+    Demo
+  </stache-page-anchor>
+  <div class="stache-demo-page-result">
+    <stache-image imageSource="~/assets/hero-sample-1.jpg"></stache-image>
```
Comment (Blackbaud-BrandonJones) [inline src/app/components/image/index.html]: You're going to need to add the no width space thing here, or builder will change this ~/assets/  to the actual path.

Comment (Blackbaud-BrandonJones) [inline src/app/components/image/index.html]: nvm, I see now you are actually putting in the image...  RIP in peas

Comment (blackbaud-brandonhare) [inline src/app/components/image/index.html]: LOL

File: src/app/components/include/index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<app-nav></app-nav>
+<stache
+  pageTitle="Include">
+  <stache-page-summary>
+    The Include component allows you to single source content and reuse it throughout your documentation.
```
Comment (Blackbaud-BrandonJones) [inline src/app/components/include/index.html]: `allows you to single source content` is kind of hard to read at first?  I had to read it a couple time to understand it.

File: src/app/components/include/index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  </div>
+
+  <stache-page-anchor>
+    Code
+  </stache-page-anchor>
+  <div class="stache-demo-code">
+    <sky-tabset>
+      <sky-tab tabHeading="code-demo-component.html">
+        <stache-code-block languageType="markup">
+          &lt;stache-include fileName="example.html"&gt;
```
Comment (Blackbaud-BrandonJones) [inline src/app/components/include/index.html]: `&lt;` shouldn't be needed anymore inside the code block?

File: src/app/components/include/index.html (hunk 2 above)
Comment (blackbaud-brandonhare) [inline src/app/components/include/index.html]: It's a common documentation idea. I'll see what @blackbaud-johnly thinks of it.

File: src/app/components/include/index.html (hunk 3 above)
Comment (blackbaud-brandonhare) [inline src/app/components/include/index.html]: It does for this. It will render that as an includes otherwise.

Comment (Blackbaud-BrandonJones) [inline src/app/components/include/index.html]: oh weird... `pluginception` definitely something we need to keep in mind :D

Comment (blackbaud-brandonhare) [inline src/app/components/include/index.html]: Yeah for sure. I suppose it depends on the order the plugins run. If the code block one runs first, it should escape it. I just figured I'd pre escape the tags just in case, since it was such a small sample.

File: src/app/components/include/index.html (hunk 2 above)
Comment (Blackbaud-BrandonJones) [inline src/app/components/include/index.html]: The idea is, but the phrasing is a bit complex... or maybe it's just me.

Comment (blackbaud-johnly) [inline src/app/components/include/index.html]: I see this PR already merged, but I agree about the wording being a little awkward. I think it's fine to leave as-is for now, but we can probably improve the text. I think the main reason it's hard to follow at first is that you expect a verb after "to" but you get "single" instead. The simplest fix is to hyphenate "single-source" to indicate that it is a compound verb. ... That said, I think it might be good to rework this beyond the hyphen. Ultimately, I'd suggest something like "The include component supports single source documentation. It allows you to store content in a source location and then reuse it throughout your documentation." Or something along those lines.
